autopush/main.py

```diff
@@ -196,6 +196,7 @@ def make_settings(args, **kwargs):
             pingConf = None
 
     return AutopushSettings(
+        reactor,
```

Comment:
Updated. I passed it around per the [Twisted docs](https://twistedmatrix.com/documents/current/api/twisted.internet.reactor.html)...

> New application code should prefer to pass and accept the reactor as a parameter where it is needed, rather than relying on being able to import this module to get a reference.

But we're not using the reactor in the tests.